dynamic_programming/fizz_buzz.py

```diff
@@ -0,0 +1,11 @@
+#https://en.wikipedia.org/wiki/Fizz_buzz#Programming
+number = 1
+out = ""
+iterations = 100
+
+while number <= int(iterations):
+  if number % 3 == 0: out += "Fizz"
+  if number % 5 == 0: out +="Buzz"
+  if out == "": out = number
```

Comment:
```suggestion
  if out == "":
      out = number
```
Makes code more readable